jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -403,6 +417,7 @@ def multipass_pipeline(jobs) {
                     SCRIPT_DIR,
                     server_era)
             step030_customize_settings(SCRIPT_DIR, job['puppet_settings'])
+            step035_customize_hocon_settings(SCRIPT_DIR, job['hocon_settings'])
```

Comment:
Ah, I forgot we disable the puppet agent, that makes sense. I'm not sure where would be the appropriate place to move this step to